test/elixir/lib/couch.ex

```diff
@@ -80,6 +80,10 @@ defmodule Couch do
   @request_timeout 60_000
   @inactivity_timeout 55_000
 
+  def base_url() do
```

Comment:
Nice, I should have done this during the porting.

Reply:
Thank you!